### ISSUE TITLE
Problem with microphone

### ISSUE THREAD

**Issue by jacopobonomi** [flutterjanus/flutter_janus_client#135]
I can't understand why in the audio plugin you need to enable the microphone just for receive it, in the previous realase there was a flag to specify whether to send audio.
Besides that, it is not clear to me why the initializeMediaDevices function is mandatory to receive audio.

**Comment by shivanshtalwar0** [comment]
I don't fully understood your question can you rephrase it pictures and better context like code snippet

**Comment by shivanshtalwar0** [comment]
Closing due to inactivity
